core/modules/czar/python/app.py

```diff
@@ -340,7 +339,8 @@ def __init__(self, query, context, setSessionId, resultName=""):
         pass
 
     def _reportError(self, chunkId, code, message):
-        logger.dbg("reporting", chunkId, code, message)
+        # TODO remove lsst/log wrapper
+        logger.dbg("Reporting:\n\tchunkId=[%s]\n\tcode=[%s]\n\tmessage=[%s]\n" % (chunkId, code, message))
```

Comment:
Does this need to be multi-line message?
Use lazy formatting for logging (I think logger wrapper should be able to support it).